docs/source/onedal/build_app/build-application.rst

```diff
@@ -50,26 +50,15 @@ Applications on Linux* OS
           :class: longtable
 
           * -
```

Comment:
@aepanchi  - i guess we don't even need separation at this level as this is just single version for multithreaded libs?

Reply:
Agree